core/server/web/api/v2/admin/uploadHelper.js

```diff
@@ -0,0 +1,27 @@
+const shared = require('../../../shared');
```

Comment:
If we move mutler initialization logic into this module, then it's interface could be used just like a regular middleware and it can be moved to the rest of middlewares in `/shared/middlewares/upload.js`

Reply:
Yes, sounds good! So we combine uploadHelper.js with upload-clear.js in upload.js and we have one file for the whole logic! Awesome😁